process/block/block_processor.go

```diff
@@ -1,18 +1,123 @@
 package block
 
 import (
+	"github.com/ElrondNetwork/covalent-indexer-go"
+	"github.com/ElrondNetwork/covalent-indexer-go/process"
+	"github.com/ElrondNetwork/covalent-indexer-go/process/utility"
 	"github.com/ElrondNetwork/covalent-indexer-go/schema"
+	"github.com/ElrondNetwork/elrond-go-core/core"
+	"github.com/ElrondNetwork/elrond-go-core/core/check"
 	"github.com/ElrondNetwork/elrond-go-core/data"
+	erdBlock "github.com/ElrondNetwork/elrond-go-core/data/block"
+	"github.com/ElrondNetwork/elrond-go-core/data/indexer"
+	"github.com/ElrondNetwork/elrond-go-core/marshal"
 )
 
-type blockProcessor struct{}
+const DefaultProposerIndex = int64(0)
```

Comment:
`ProposerIndex` instead of DefaultProposerIndex --- all the time proposer index will be 0

Reply:
Changed it.